sensing/autoware_pointcloud_preprocessor/src/filter.cpp

```diff
@@ -359,19 +359,12 @@ bool autoware::pointcloud_preprocessor::Filter::convert_output_costly(
 void autoware::pointcloud_preprocessor::Filter::faster_input_indices_callback(
   const PointCloud2ConstPtr cloud, const PointIndicesConstPtr indices)
 {
-  // TODO(knzo25): This first branch is to give temporary compatibility with old perception data.
-  // Should be deleted soon
-  if (utils::is_data_layout_compatible_with_point_xyzi(*cloud)) {
-    RCLCPP_ERROR_THROTTLE(
-      get_logger(), *get_clock(), 10000,
-      "The pointcloud layout is compatible with PointXYZI. You may be using legacy "
-      "code/data. Continue at your own risk");
-  } else if (
+  if (
     !utils::is_data_layout_compatible_with_point_xyzircaedt(*cloud) &&
     !utils::is_data_layout_compatible_with_point_xyzirc(*cloud)) {
     RCLCPP_ERROR(
       get_logger(),
-      "The pointcloud layout is not compatible with PointXYZIRCAEDT/PointXYZIRC. Aborting");
+      "The pointcloud layout is not compatible with PointXYZIRCAEDT not PointXYZIRC. Aborting");
```

Comment:
@Shin-kyoto @kminoda cc @knzo25 
Since some node in perception pipeline did not strictly require PointXYZIRC for input such as Ground_Segmentation, compare_map_filter, is possible to remove this as before?

Reply:
@Shin-kyoto 
Is your question resolved [here](https://star4.slack.com/archives/C03S84LDJGG/p1723093872424089?thread_ts=1723091914.459219&cid=C03S84LDJGG)?

Reply:
@Shin-kyoto Yes, as internal discussion it's is clear for me now. thanks.